styles/style.css

```diff
@@ -0,0 +1,81 @@
+.container {
```

Comment:
[OPTIONAL] & [HIGHLY RECOMMENDED]

In order to follow good practices and in order for you to be easier to design you project, always is recommended to start your CSS with the following from the first line of code: 

`* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}`

You will thank me later ;)